mlflow/cli.py

```diff
@@ -53,9 +52,8 @@ def cli():
 @click.option("--experiment-name", envvar=tracking._EXPERIMENT_NAME_ENV_VAR,
               help="Name of the experiment under which to launch the run. If not "
                    "specified, 'experiment-id' option will be used to launch run.")
-@click.option("--experiment-id", envvar=tracking._EXPERIMENT_ID_ENV_VAR, type=click.INT,
-              help="ID of the experiment under which to launch the run. Defaults to %s" %
-                   Experiment.DEFAULT_EXPERIMENT_ID)
+@click.option("--experiment-id", envvar=tracking._EXPERIMENT_ID_ENV_VAR, type=click.STRING,
+              help="ID of the experiment under which to launch the run. Defaults to %s" % None)
```

Comment:
This should still be DEFAULT_EXPERIMENT_ID if you're going to make that value "0". But I think empty string or None make more sense.